istioctl/pkg/auth/upgrade.go

```diff
@@ -41,12 +46,13 @@ type WorkloadLabels map[string]string
 type Upgrader struct {
 	IstioConfigStore model.ConfigStore
 	K8sClient        *kubernetes.Clientset
+	ServiceFiles     []string
 	// RoleNameToWorkloadLabels maps the ServiceRole name to the map of service name to the workload labels
 	// that the service pointing to.
 	// We could have remove the `service` key layer to make it ServiceRole name maps to workload labels.
 	// However, this is needed for unit tests.
 	RoleNameToWorkloadLabels map[string]ServiceToWorkloadLabels
-	RbacFile                 string
+	V1PolicyFile             string
```

Comment:
Why changing the name? Are we planning on using this `upgrader` for authN as well?

Reply:
Yes, I'm planning to make the `upgrade` support both AuthN and AuthZ.

/cc @diemtvu